modules/tinymce/src/core/main/ts/selection/ElementSelection.ts

```diff
@@ -93,6 +98,20 @@ const getNode = (root: Element, rng: Range): Element => {
         return startContainer;
       }
     }
+
+    if (startContainer?.nodeType === 3 && endContainer?.nodeType === 1) {
+      if (endContainer.childElementCount === 2) {
+        return endContainer.firstChild;
+      }
+      if (startOffset > endOffset || isAnchorPrevSibling(startContainer)) {
+        return startContainer.nextSibling;
+      }
+    }
+
+    // IE Handle double click an anchor.
+    if (startContainer?.nodeType === 1 && endContainer?.nodeType === 3) {
+      return startContainer.firstChild;
```

Comment:
Same here with NodeType and firstChild.

Btw, if you have `|<p>&nbsp;</p><p>a|b</p>`, Selection.getNode will no longer return "_the currently selected element or the common ancestor element for both start and end of the selection_" so looks like you would want to do something else here.

Reply:
updated